src/libraries/Common/src/Microsoft/Win32/SafeHandles/SafeProcessHandle.Windows.cs

```diff
@@ -16,7 +16,12 @@
 
 namespace Microsoft.Win32.SafeHandles
 {
-    public sealed partial class SafeProcessHandle : SafeHandleZeroOrMinusOneIsInvalid
+#if SYSTEM_DIAGNOSTICS_PROCESS
```

Comment:
These changes won't be necessary once you address my other feedback.

Reply:
Done!